functions/public/Invoke-WPFUIElements.ps1

```diff
@@ -0,0 +1,315 @@
+                            [System.Object]$Sender = $args[0]
+                            Invoke-WPFToggle $Sender.name
+                        })
+                    }
+
+                    "ToggleButton" {
+                        $toggleButton = New-Object Windows.Controls.ToggleButton
+                        $toggleButton.Name = $entryInfo.Name
+                        $toggleButton.Name = "WPFTab" + ($stackPanel.Children.Count + 1) + "BT"
+                        $toggleButton.HorizontalAlignment = "Left"
+                        $toggleButton.Height = $theme.TabButtonHeight
+                        $toggleButton.Width = $theme.TabButtonWidth
+                        $toggleButton.Background = $theme.ButtonInstallBackgroundColor
+                        $toggleButton.Foreground = [Windows.Media.Brushes]::White
+                        $toggleButton.FontWeight = [Windows.FontWeights]::Bold
+
+                        $textBlock = New-Object Windows.Controls.TextBlock
+                        $textBlock.FontSize = $theme.TabButtonFontSize
+                        $textBlock.Background = [Windows.Media.Brushes]::Transparent
+                        $textBlock.Foreground = $theme.ButtonInstallForegroundColor
+
+                        $underline = New-Object Windows.Documents.Underline
+                        $underline.Inlines.Add($entryInfo.name -replace "(.).*", "`$1")
+
+                        $run = New-Object Windows.Documents.Run
+                        $run.Text = $entryInfo.name -replace "^.", ""
+
+                        $textBlock.Inlines.Add($underline)
+                        $textBlock.Inlines.Add($run)
+
+                        $toggleButton.Content = $textBlock
+
+                        $stackPanel.Children.Add($toggleButton) | Out-Null
+
+                        $sync[$entryInfo.Name] = $toggleButton
+                    }
+
+                    "Combobox" {
+                        $horizontalStackPanel = New-Object Windows.Controls.StackPanel
+                        $horizontalStackPanel.Orientation = "Horizontal"
+                        $horizontalStackPanel.Margin = "0,5,0,0"
+
+                        $label = New-Object Windows.Controls.Label
+                        $label.Content = $entryInfo.Content
+                        $label.HorizontalAlignment = "Left"
+                        $label.VerticalAlignment = "Center"
+                        $label.FontSize = $theme.ButtonFontSize
+                        $horizontalStackPanel.Children.Add($label) | Out-Null
+
+                        $comboBox = New-Object Windows.Controls.ComboBox
+                        $comboBox.Name = $entryInfo.Name
+                        $comboBox.Height = $theme.ButtonHeight
+                        $comboBox.Width = $theme.ButtonWidth
+                        $comboBox.HorizontalAlignment = "Left"
+                        $comboBox.VerticalAlignment = "Center"
+                        $comboBox.Margin = "5,5"
```

Comment:
@MyDrift-user it's interesting that in `themes.json` file you've custom margins for each theme... but you don't seem to use it.. Is there any reason behind this? or is it a simple oversight when you've coded this part?